<launch>
<arg name="choose_web_cam" default="True"/>
<param name="tracking_method" value="handtracking"/>
<param name="tracker_updated" type="bool" value="true"/>
<group if="$(arg choose_web_cam)">
<param name= "usb_cam/pixel_format" value="yuyv"/>
<param name= "camera_topic" value="/usb_cam/image_raw"/>
<node pkg="usb_cam" name="usb_cam" type="usb_cam_node"/>
</group>
<group unless="$(arg choose_web_cam)">
<include file="$(find realsense)/realsense_r200_launch.launch"/>
<param name= "camera_topic" value="/camera/color/image_raw"/>
</group>
<node pkg="cmt_tracker" name="face_locator_node" type="face_locator_node"/>
<node pkg="cmt_tracker" name="cmt_tracker_node" type="cmt_tracker_node"/>
</launch>
